SonarQube.TeamBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -78,7 +84,9 @@ public ProcessedArgs(string key, string name, string version, bool installLoader
         /// </summary>
         public IAnalysisPropertyProvider AggregateProperties { get { return this.aggProperties; } }
 
-        public IAnalysisPropertyProvider LocalProperties { get { return this.cmdLineProperties; } }
+        public IAnalysisPropertyProvider CmdLineProperties { get { return this.cmdLineProperties; } }
```

Comment:
Just FYI you can also use a new syntax which make these readonly properties easier to write.
Before:
`public IAnalysisPropertyProvider CmdLineProperties { get { return this.cmdLineProperties; } }`
After:
`public IAnalysisPropertyProvider CmdLineProperties => this.cmdLineProperties;`